packages/relay/src/lib/eth.ts

```diff
@@ -900,13 +900,16 @@ export class EthImpl implements Eth {
       throw this.genericErrorHandler(e);
     }
 
+    const parsedTx = Precheck.parseTxIfNeeded(transaction);
```

Comment:
This means the transaction gets parsed twice.
Inside `this.precheck.sendRawTransactionCheck` and now here.
Should we have `this.precheck.sendRawTransactionCheck` return it so it's done once or what other options do we have?

Reply:
or we could parsed it before and then send the transaction already parsed, I like more that option, since right now there are no other usages for that method, so little refactor, and the method purpose does not get abused